content-security-policy/webrtc/webrtc.js

```diff
@@ -0,0 +1,62 @@
+
+// Creates two RTCPeerConnection and tries to connect them. Returns
+// "allowed" if the connection is permitted, "blocked" if it is
+// blocked on both sides and "inconsistent" in the even that the
+// result is the same on both sides (should never happen).
+async function tryConnect() {
+    const iceServers = [{urls: "stun:stun.l.google.com:19302"}];
```

Comment:
I'm not sure what the code below does, but I think WPTs should not depend on external services like google.com

Reply:
It seems like in principle we should be able to avoid this, since the stun server is typically for NAT traversal. If I just set this to an empty list though I get:

```
  │   → Unhandled rejection: No remoteDescription.
  │ 
  └ pcFailed/pc.onicecandidate@http://web-platform.test:8000/content-security-policy/webrtc/webrtc.js:17:1
```

...on two of the tests (instead of the ufrag error). Maybe @alvestrand or @jan-ivar can shed some on the right way to do this; I feel like I'm kindof fumbling around piecing this together.

Reply:
Removed.